src/modules/PublicLab.MainImageModule.js

```diff
@@ -127,6 +128,7 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
       progressall: function (e, data) {
 
         var progress = parseInt(data.loaded / data.total * 100, 10);
+        dragImageI.innerHTML = "";
```

Comment:
Could you please add a comment here, so x months later down the road we know why we did it? Thank you.